lib/ex_unit/lib/ex_unit.ex

```diff
@@ -251,6 +251,8 @@ defmodule ExUnit do
     * `:failures_manifest_file` - specifies a path to the file used to store failures
       between runs;
 
+    * `:skip_failures_manifest_file` - skips writing to the failures manifest file;
```

Comment:
This should probably have a better name, because we only skip writing the file, not reading it